extensions/parquet/table/build.gradle

```diff
@@ -51,6 +48,9 @@ dependencies {
             project(path: ':test-configs')
     Classpaths.inheritSlf4j(project, 'slf4j-simple', 'testRuntimeOnly')
 
+    implementation project(':plugin-trackedseekablechannel')
```

Comment:
I think this could be declared as runtimeOnly (given changes suggested elsewhere to `io.deephaven.parquet.table.ParquetTableWriter#getParquetFileWriter`)